guides/release/accessibility/learning-resources.md

```diff
@@ -1,31 +1,33 @@
-- [Web Content Accessibility Guidelines(WCAG) 2.1](https://www.w3.org/TR/WCAG21/)
-- [Accessible Rich Internet Applications (ARIA) 1.1](https://www.w3.org/TR/wai-aria-1.1/) 
-- [How to meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)
+*Note: La plupart des liens ci-dessous redirigent vers des articles écrits en Anglais
 
+- [Documentation Web MDN: Accessibilité](https://developer.mozilla.org/fr/docs/Learn/Accessibility)
 
-## Practical Resources
+- [Using ARIA](https://www.w3.org/TR/using-aria/) : un guide pratique pour ajouter aux éléments HTML des informations accessibles.
+- [Web Content Accessibility Guidelines(WCAG) : Directives du contenu web sur l'accessibilité 2.1](https://www.w3.org/TR/WCAG21/)
+- [Accessible Rich Internet Applications (ARIA) : Applications Internet Riches et Accessibles 1.1](https://www.w3.org/TR/wai-aria-1.1/)
+- [How to Meet WCAG (Quick Reference)](https://www.w3.org/WAI/WCAG21/quickref/)
 
-- [Accessibility Insights](https://accessibilityinsights.io/)- Use FastPass to find common, high-impact issues 
-- [aXe extension for Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
-- [Accessibility Support](https://a11ysupport.io/)- Find out if your code will work with assistive tech
+## Ressources pratiques
+
+- [Accessibility Insights](https://accessibilityinsights.io/)- Utiliser FastPass pour trouver des problèmes communs à fort impact
+- [Extension aXe pour Chrome](https://chrome.google.com/webstore/detail/axe/lhdoppojpmngadmnindnejefpokejbdd)
+- [Accessibility Support](https://a11ysupport.io/)- Comprendre si son code marche avec les technologies d'assistance
 - [How and where to report accessibility bugs](https://www.digitala11y.com/how-where-to-report-accessibility-bugs/)
 
 
-### Design 
+### Design
 
-- [ColorSafe](http://colorsafe.co/)- Empowering designers with beautiful and accessible color palettes based on WCAG Guidelines of text and background contrast ratios
-- [Accessible Color Palette Builder](https://toolness.github.io/accessible-color-matrix/)
+- [ColorSafe](http://colorsafe.co/)- Appuyer les designers en proposant de magnifiques palettes de couleurs accessibles, basées sur les directives WCAG, avec ratio de couleurs de contraste de textes et arrières plans.
+- [Créateur de palettes de couleur accessibles](https://toolness.github.io/accessible-color-matrix/)
 
 
-### Writing
+### Écriture
 
-- [Plain Language](https://plainlanguage.gov/) - Learn how to write in a way that makes it easier for people to read and understand. 
+- [Langage clair et simple](https://plainlanguage.gov/) - Apprendre comment écrire d'une manière accessible et inclusive pour une compréhension de lecture plus facile.
 
 
-### Other Useful Articles
+### Autres articles utiles
 
 - [The difference between keyboard and screen reader navigation](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/)
```

Comment:
You forgot to translate this 😅 .

```suggestion
- [La différence entre navigation au clavier et navigation avec un lecteur d'écran](https://tink.uk/the-difference-between-keyboard-and-screen-reader-navigation/)
```

Reply:
@BlueCutOfficial Actually suggested to keep the name in English, so I applied her commit 😅 Let me know @KamiKillertO or Blue what is best!

Reply:
Ok. It's fine like this 👍